### PR TITLE
E0d/common refactor hackathon

### PR DESCRIPTION
**Pull request by e0d**
@maxrothman

### CODE REVIEW

**Comment by fredsmith** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:

**Comment by maxrothman** [comment]
Looks like this caught 2f392cf, but not https://github.com/edx/configuration/commit/818d8dc1b71bae52f5c15d785858a9678b905b1b, which reverts it. That'll have to be added before https://github.com/edx/configuration/pull/2349 merges.

File: playbooks/roles/analytics_api/defaults/main.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ ANALYTICS_API_DATABASES:
     NAME: '{{ ANALYTICS_API_DEFAULT_DB_NAME }}'
     USER: 'api001'
     PASSWORD: 'password'
-    HOST: 'localhost'
+    HOST: '{{ analytics_api_db_default_hostname }}'
```
Comment (feanil) [inline playbooks/roles/analytics_api/defaults/main.yml:32]: The case of this doesn't match the var names below which are in all caps.

File: playbooks/roles/analytics_api/defaults/main.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ ANALYTICS_API_DATABASES:
     NAME: '{{ ANALYTICS_API_REPORTS_DB_NAME }}'
     USER: 'reports001'
     PASSWORD: 'password'
-    HOST: 'localhost'
+    HOST: '{{ analytics_api_db_reports_hostname }}'
```
Comment (feanil) [inline playbooks/roles/analytics_api/defaults/main.yml:40]: same problem as `analytics_api_db_default_hostname`

File: playbooks/roles/analytics_api/defaults/main.yml (hunk 1 above)
Comment (maxrothman) [inline playbooks/roles/analytics_api/defaults/main.yml:32]: This change was reverted (mentioned above), but somehow got missed in this PR.

**Pull request merged**